### ISSUE TITLE
search fails: Make sure input fields are valid

### ISSUE THREAD

**Issue by jonassmedegaard** [out-of-cheese-error/rust-hypothesis#10]
Running `hypothesis-client profile user` succesfully reports my account info, but `hypothesis-client annotations search` fails like this:
```
Error: 
   0: Make sure input fields are valid
   1: Status: 
      Reason: 

Location:
   /build/reproducible-path/rust-hypothesis-0.11.1/src/cli.rs:223
```

**Comment by Ninjani** [comment]
Thanks, fixed in PR #11. I'll make a new release

**Comment by jonassmedegaard** [comment]
Yes, I can confirm that the fix works. Great!
